java/java-guestbook/frontend/src/main/resources/templates/home.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Guestbook</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" th:href="@{/style.css}" type="text/css">
+    <link rel="icon" th:href="@{/favicon.png}" type="image/x-icon"/>
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
```

Comment:
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

Reply:
FYI @simonz130 

we're reusing html templates made by designers, so idk if we want to make changes that would have to be replicated between all languages.